Xamarin.Forms.Platform.iOS/Renderers/TableViewModelRenderer.cs

```diff
@@ -100,6 +105,20 @@ public override string TitleForHeader(UITableView tableView, nint section)
 			return View.Model.GetSectionTitle((int)section);
 		}
 
+		protected void WireUpForceUpdateSizeRequested(ICellController cell, UITableViewCell nativeCell, UITableView tableView)
```

Comment:
Is this code addition necessary?

It seems like the code here
https://github.com/xamarin/Xamarin.Forms/blob/14af670f5a200d164284fa4f0b3e4f7ce03b53df/Xamarin.Forms.Platform.iOS/Cells/CellRenderer.cs#L60

Takes care of the Reload.  I commented this addition here out and the UI Test still passed because the ReloadRows inside CellRenderer takes care of it.

The addition to *Cell.cs* seems to be enough